newIDE/app/src/MainFrame/EditorContainers/HomePage/HomePageMenu.js

```diff
@@ -2,37 +2,22 @@
 import * as React from 'react';
 import { Trans } from '@lingui/macro';
 import { Column, Line } from '../../../UI/Grid';
-import { Drawer } from '@material-ui/core';
-import { useResponsiveWindowWidth } from '../../../UI/Reponsive/ResponsiveWindowMeasurer';
+import Drawer from '@material-ui/core/Drawer';
 import IconButton from '../../../UI/IconButton';
-import DoubleChevronArrowRight from '../../../UI/CustomSvgIcons/DoubleChevronArrowRight';
 import VerticalTabButton from '../../../UI/VerticalTabButton';
 import DoubleChevronArrowLeft from '../../../UI/CustomSvgIcons/DoubleChevronArrowLeft';
 import PickAxeIcon from '../../../UI/CustomSvgIcons/PickAxe';
 import SchoolIcon from '../../../UI/CustomSvgIcons/School';
 import GoogleControllerIcon from '../../../UI/CustomSvgIcons/GoogleController';
 import WebIcon from '../../../UI/CustomSvgIcons/Web';
-import Sun from '../../../UI/CustomSvgIcons/Sun';
-import Store from '../../../UI/CustomSvgIcons/Store';
+import SunIcon from '../../../UI/CustomSvgIcons/Sun';
+import StoreIcon from '../../../UI/CustomSvgIcons/Store';
 import Preferences from '../../../UI/CustomSvgIcons/Preferences';
 import GDevelopGLogo from '../../../UI/CustomSvgIcons/GDevelopGLogo';
 import GDevelopThemeContext from '../../../UI/Theme/GDevelopThemeContext';
-import Paper from '../../../UI/Paper';
+import HomePageMenuBar from './HomePageMenuBar';
```

Comment:
should this be called a bar if the menu is on the left on desktop? (don't have a better name yet though...)

Reply:
I think it's fine, the task *bar* on Windows used to be able to live on the left